src/main/java/org/mitre/synthea/export/rif/tools/BB2RIFBeneSplitter.java

```diff
@@ -0,0 +1,89 @@
+package org.mitre.synthea.export.rif.tools;
+
+import com.fasterxml.jackson.databind.MappingIterator;
+import com.fasterxml.jackson.databind.SequenceWriter;
+import com.fasterxml.jackson.dataformat.csv.CsvMapper;
+import com.fasterxml.jackson.dataformat.csv.CsvSchema;
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Path;
+import java.util.HashMap;
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+/**
+ * Utility for converting from original BB2 RIF 3 file bene output to one bene file per year.
+ */
+public class BB2RIFBeneSplitter {
+
+  static final String[] inputFiles = {
+    "beneficiary.csv",
+    "beneficiary_interim.csv",
+    "beneficiary_final.csv"
+  };
+
+  static final String[] yearFields = {
+    "RFRNC_YR", // BFD
+    "BENE_ENROLLMT_REF_YR" // CCW
+  };
+
+  /**
+   * Split original 3 bene files into one file per year.Read in original beneficiary.csv,
+   * beneficiary_interim.csv and beneficiary_final.csv from the current directory and output one
+   * or more beneficiary_YYYY.csv files where YYYY is the numerical year.
+   * Works with either BFD or CCW format files and preserves whatever format exists in the output
+   * files
+   * @param args unused
+   * @throws java.io.IOException if something goes wrong
+   */
+  public static void main(String[] args) throws IOException {
+    if (args.length != 1) {
+      System.out.println("Usage ./gradlew rifBeneSplit -Pargs=\"inputDir\"");
+      System.exit(-1);
+    }
+    Map<String, SequenceWriter> writers = new HashMap<>();
+    for (String inputFileName: inputFiles) {
+      File inputFile = Path.of(args[0]).resolve(inputFileName).toFile();
```

Comment:
Not an issue worth stopping this pull request, but this path code was not working for me when the path contained whitespace (possibly an escaping issue on the command line) or when the path contained a tilde (`~`) to the user's home folder.